### DISCUSSION TITLE
When to start learning

### DISCUSSION THREAD

**Discussion by avisabpre** [microsoft/ML-For-Beginners#121] [Q&A] [answered]
Is this already ongoing and should i wait for next set of freshers to start. How do i begin learning.

**Comment by jasonxavierdsouza** [comment]
This is pre-recorded video

**Reply by jlooper** [reply]
The videos accompany the written lessons. This is a self-guided course.

**Comment by jlooper** [comment, marked as answer]
> Is this already ongoing and should i wait for next set of freshers to start. How do i begin learning.

Please read the README to get instructions on how to learn. This is a self-driven course.
